target/linux/realtek/dts/rtl8382_zyxel_gs1900-24-v1.dts

```diff
@@ -1,7 +1,6 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 
-#include "rtl8380_zyxel_gs1900.dtsi"
-#include "rtl8380_zyxel_gs1900_gpio.dtsi"
+#include "rtl8382_zyxel_gs1900-24.dtsi"
 
 / {
 	compatible = "zyxel,gs1900-24-v1", "realtek,rtl838x-soc";
```

Comment:
If the v2 gets `model = ... rev. B1`, perhaps this could be updated to say `rev. A1`